modules/mempool.nix

```diff
@@ -0,0 +1,331 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services = {
+    mempool = {
+      enable = mkOption {
```

Comment:
Is there a reason to not use a `mkEnableOption` here?

Reply:
Mempool has an extended, multi-sentence description text which doesn't fit the description template of `mkEnableOption`.